### ISSUE TITLE
Train on my own dataset. How do I mark up my own dataset with people?

### ISSUE THREAD

**Issue by sergeya3ro** [qubvel/segmentation_models.pytorch#134]
I'm new in machine learning, so there are few things I can't solve.
I have run the example from this source successfully (https://github.com/qubvel/segmentation_models.pytorch/blob/master/examples/cars%20segmentation%20(camvid).ipynb).
Now I want to improve the train results on my own dataset with people (or 'pedestrian'). So I have some questions.
1) But i have some problems with this. I figured out that I can't just make a binary mask for my class to each image in my dataset. From Example I only noticed that the mask color differs depending on the class, but I didn't understand how the program links this color to the class index.What is the direct relationship? 
2) I searched another one implementation of Unet with pretrained VGG-16_bn weights (https://app.supervise.ly/models/my/13155/overview). But these weights are quite different from the weights from the example in size (250 mb vs 540 mb) Which of them is better? Can I somehow use the VGG-16 (250 mb) as an encoder from supervisely? 

I hope my questions will be correctly understand and I hope that you will help.
